libs/parsers/src/lib/swagger/parser.ts

```diff
@@ -288,7 +288,7 @@ export class SwaggerParser {
     return map(operations, 'operationId');
   }
 
-  getOperationByOperationId(operationId: string): Operation {
+  getOperationByOperationId(operationId: string): Operation | undefined {
```

Comment:
This can return `undefined` since `getPropertyValue` has a return type of `any` and can potentially just default to `undefined`.

The rest of the changes in this PR are to handle the possible `undefined` return value from this method.

![image](https://github.com/Azure/LogicAppsUX/assets/1350074/5a7a8453-c6b1-4c60-a098-dbb7dd89026e)